src/org/parosproxy/paros/extension/option/OptionsConnectionPanel.java

```diff
@@ -854,7 +858,18 @@ private JCheckBox getCheckBoxSingleCookeRequestHeader() {
         return checkBoxSingleCookieRequestHeader;
     }
 
-    private ProxyExcludedDomainsMultipleOptionsPanel getProxyExcludedDomainsPanel() {
+    public JCheckBox getCheckBoxHttpStateEnabled() {
+    	if (checkBoxHttpStateEnabled == null) {
+    		checkBoxHttpStateEnabled = new JCheckBox(Constant.messages.getString("conn.options.httpStateEnabled"));
+    	}
+		return checkBoxHttpStateEnabled;
+	}

```

Comment:
Seems to be something weird with the indentation here, I'm guessing there's mixed spaces and tabs.